.github/workflows/UnitTest.yml

```
@@ -0,0 +1,61 @@
+name: Unit test
+
+on:
+  create:
+    tags:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+      - '.github/workflows/TagBot.yml'
+  pull_request:
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+      - '.github/workflows/TagBot.yml'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # allow-failure is not supported yet
+      # https://github.com/actions/toolkit/issues/399
+      fail-fast: false
+      matrix:
+        julia-version: ['1.0', '1', 'nightly']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        julia-arch: [x64]
+        # only test one 32-bit job
+        include:
+          - os: ubuntu-latest
+            julia-version: '1'
+            julia-arch: x86
+
+    steps:
+      - uses: actions/checkout@v1.0.0
```

No reason that I know of to use the old version. See <https://github.com/marketplace/actions/checkout>:

```suggestion
      - uses: actions/checkout@v2
```